UnityProjects/MRTKDevTemplate/Assets/Scripts/InspectorDrivenDialog.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Serialization;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UX
```

Comment:
Do we want this in the examples/samples/demos namespace? Or is this intended to go into a UX package at some point?

Reply:
If/when we want to graduate it, it would go into uxcore.

Reply:
Fixed namespace.